ethcore/src/json_tests/chain.rs

```diff
@@ -172,14 +188,23 @@ mod block_tests {
 	declare_test!{BlockchainTests_GeneralStateTest_stRandom2, "BlockchainTests/GeneralStateTests/stRandom2/"}
 	declare_test!{BlockchainTests_GeneralStateTest_stRecursiveCreate, "BlockchainTests/GeneralStateTests/stRecursiveCreate/"}
 	declare_test!{BlockchainTests_GeneralStateTest_stRefundTest, "BlockchainTests/GeneralStateTests/stRefundTest/"}
-	declare_test!{BlockchainTests_GeneralStateTest_stReturnDataTest, "BlockchainTests/GeneralStateTests/stReturnDataTest/"}
+	// todo[dvdplm]: 1 test fails. Single test. Have not investigated. Ignored in currents.json.
```

Comment:
Do we have an issue number for this?

Reply:
Same for the remaining `TODOS`, insert issue number if we got it

Reply:
Probably best if I remove the todos now that we have real issues filed and linked to this PR?

Reply:
yeah, that would the best